packages/transport-webrtc/package.json

```
@@ -68,17 +70,16 @@
     "uint8arrays": "^4.0.4"
   },
   "devDependencies": {
-    "@chainsafe/libp2p-yamux": "^4.0.1",
-    "@libp2p/interface-compliance-tests": "^3.0.0",
-    "@libp2p/peer-id-factory": "^2.0.0",
-    "@libp2p/websockets": "^6.0.0",
+    "@chainsafe/libp2p-yamux": "^4.0.2",
+    "@libp2p/peer-id-factory": "^2.0.4",
+    "@libp2p/websockets": "^6.0.3",
```

Not necessary as these are monorepo siblings and this change will be undone by `aegir check-project`.

```suggestion
    "@libp2p/peer-id-factory": "^2.0.0",
    "@libp2p/websockets": "^6.0.0",
```